backend/amd64/arch.ml

```diff
@@ -263,6 +263,8 @@ let operation_can_raise = function
   | Istore_int (_, _, _) | Ioffset_loc (_, _)
   | Iprefetch _ -> false
 
+let operation_allocates _ = false
```

Comment:
I know I am being annoying, but using
a catch-all pattern essentially defeats
the purpose of the new function: the
compiler will not warn us when we add
new instructions.

Reply:
good point, thanks! fixing